src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -340,17 +340,12 @@ public UserDetails loadUserByUsername(String username) throws UsernameNotFoundEx
 							throw new UsernameNotFoundException(username);
 						}
 						LOGGER.fine("loadUserByUsername in createSecurityComponents called, user: " + u);
-						List<UserProperty> props = u.getAllProperties();
-						LOGGER.fine("loadUserByUsername in createSecurityComponents called, number of props: " + props.size());
+						OicUserProperty oicProp = u.getProperty(OicUserProperty.class);
 						GrantedAuthority[] auths = new GrantedAuthority[0];
```

Comment:
I'd like to suggest that replacing the line 343 and 345. But this comment does not block the review process.